src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -173,6 +173,14 @@ public void setDesiredState(SwerveModuleState desiredState) {
         desiredSpeedLog.append(desiredState.speedMetersPerSecond);
     }
 
+    public void park(boolean reversed) {
```

Comment:
There is an assumption that the drive motors are stopped. If the last command running on the SwerveSubsystem was teleop, this is true because stopModules is called at the end. I'm not sure if we can assume this though.  Maybe call stop() first?

Reply:
ok added a stop()